ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -102,15 +102,7 @@ compute:
 - name: "worker"
   replicas: 0
 platform:
-  vsphere:
-    vcenter: "${vsphere_url}"
-    datacenter: "${vsphere_datacenter}"
```

Comment:
Not a 100% sure but it looks like not having the datacenter set, is causing the gather-extra [artifacts](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/22371/rehearse-22371-pull-ci-openshift-windows-machine-config-operator-release-4.9-platform-none-e2e-operator/1445780737910378496/artifacts/platform-none-e2e-operator/gather-extra/build-log.txt) not being collected.